NetKAN/UKS.netkan

```diff
@@ -25,7 +25,9 @@
   "install" : [
     {
       "file"       : "GameData/UmbraSpaceIndustries",
```

Comment:
Apply the same logic as the Karbonite comment (atm not able to quickly look at the contents of UKS.zip)